.github/workflows/scheduled-publishing.yml

```diff
@@ -0,0 +1,50 @@
+name: "Scheduled publishing"
+run-name: "Publish to internal - Scheduled"
+
+on:
+    schedule:
+    -   cron: "0 0 * * SUN"
+
+jobs:
+    publish:
+        strategy:
+            matrix:
+                package:
+                -   "dbt-athena"
+                -   "dbt-bigquery"
+                -   "dbt-postgres"
+                -   "dbt-redshift"
+                -   "dbt-snowflake"
+                -   "dbt-spark"
+            fail-fast: false
+        uses: ./.github/workflows/publish-internal.yml
+        with:
+            package: ${{ matrix.package }}
+        secrets: inherit
+
+    open-lockfile-pr:
+        runs-on: ${{ vars.DEFAULT_RUNNER }}
+        steps:
+        -   id: pr
+            shell: bash
+            run: |
+                RUN_ID=$(gh workflow run $WORKFLOW --repo $REPO)
+                echo $RUN_ID
+                gh run watch $RUN_ID
```

Comment:
You can add a `--exit-status` flag here so that the step will fail with a non-zero exit code if the called workflow does.  Not common for the called workflow though, so maybe not needed.  Just makes debugging a bit easier if it does happen.

Reply:
I initially added that more to check if the PR could be created. I thought it might fail if there are no changes in the lockfile. But I ultimately chose not to. I'm less concerned about the workflow failing due to a bug.